lib/fog/openstack/models/compute/server_group.rb

```diff
@@ -0,0 +1,22 @@
+require 'fog/openstack/models/model'
+
+module Fog
+  module Compute
+    class OpenStack
+      class ServerGroup < Fog::OpenStack::Model
+        identity  :id
+        attribute :name
+        attribute :policies, :type => :array
+        attribute :members
+
+        VALID_SERVER_GROUP_POLICIES = ['affinity', 'anti-affinity']
+
+        def self.validate_server_group_policy(policy)
+          fail ArgumentError, "#{policy} is an invalid policy... must use one of #{VALID_SERVER_GROUP_POLICIES.join(', ')}" \
+            unless VALID_SERVER_GROUP_POLICIES.include? policy
+          true
```

Comment:
That could return true / false, but then the places where that is called should raise the ArgumentError unless it was true (less efficient due to having two checks inline and also duplication of that check / raise)